stdlib/ballerina-websub/src/main/java/org/ballerinalang/net/websub/WebSubResourceDispatcher.java

```diff
@@ -88,7 +88,7 @@ static HttpResource findResource(HttpService service, HTTPCarbonMessage inboundR
                 //if the request is a GET request indicating an intent verification request, and the user has not
                 //specified an onIntentVerification resource, assume auto intent verification
                 String annotatedTopic = (service.getBalService())
-                        .getAnnotationList(WebSubSubscriberConstants.WEBSUB_PACKAGE_PATH,
+                        .getAnnotationList(WebSubSubscriberConstants.WEBSUB_PACKAGE,
```

Comment:
Shall we static import these constants as well?

Reply:
Fixed with 62b3504